src/renderers/WebGLRenderer.js

```diff
@@ -56,6 +56,8 @@ function WebGLRenderer( parameters ) {
 
 	parameters = parameters || {};
 
+	if ( ! parameters.silent ) console.log( 'Made with three.js', REVISION );
```

Comment:
When `silent` is `undefined`, `! parameters.silent` evaluates to `true`. The idea of https://github.com/mrdoob/three.js/commit/1013fbb9d8f25751c9f17a43f2ce654224e024bd was to prevent the log by default. 

> Because a few users don't want the log in their personal project,

Can we really say this? Maybe the majority of users prefers no logs by default. We have no empirical data that prove one of these statements is `true`. 

I personally think there should be no logging at all by default. Instead, I would favor a parameter that controls the log level in the library (with `silent` as default value).

Reply:
I do not like https://github.com/mrdoob/three.js/commit/1013fbb9d8f25751c9f17a43f2ce654224e024bd and have no interest in duplicating it.

@mrdoob Has made it clear for 10 years that he prefers the single line logged. That is why it has remained for 10 years. I, too, prefer the single line logged; I like to see where three.js is used in the wild.

This PR allows the users who insist on preventing the log the ability to do so.